### PR TITLE
Ignore middle-button clicks and refactor external links

### PR DESCRIPTION
**Pull request by j-a-m-l**
Suggested by @boldninja

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -122,6 +122,7 @@
   <script src="src/directives/validAmount.directive.js"></script>
   <script src="src/directives/copyToClipboard.directive.js"></script>
   <script src="src/directives/backgroundStyle.directive.js"></script>
+  <script src="src/directives/open-external.directive.js"></script>
```
Comment (spresnal) [inline client/app/index.html]: The name is inconsistent with the others here

File: client/app/src/components/layout/main-navbar.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     </md-button>
 		<img src="ark.png" alt="" height="40">
 		<md-button ng-click="$ctrl.ul.selected?$ctrl.ul.selected=null:$ctrl.ul.selected=$ctrl.ul.accounts[0]"><strong>{{$ctrl.ul.network.token}} <translate>Client</translate> - {{$ctrl.ul.clientVersion}}</strong></md-button>
-		<md-button class="new-version-button" ng-if="$ctrl.ul.clientVersion!=$ctrl.ul.latestClientVersion" onclick='require("electron").shell.openExternal("https://github.com/ArkEcosystem/ark-desktop/releases")' aria-label="New version available!">
+		<md-button class="new-version-button" ng-if="$ctrl.ul.clientVersion!=$ctrl.ul.latestClientVersion" open-external="https://github.com/ArkEcosystem/ark-desktop/releases'" aria-label="New version available!">
```
Comment (spresnal) [inline client/app/src/components/layout/main-navbar.html]: Is this missing a single quote at the start of the url?

File: client/app/src/directives/open-external.directive.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        require: '?ngModel',
+        link: function (scope, element, attrs, ctrl) {
+          angular.element(element).bind('click', function(event) {
+            const url = scope.$eval(attrs.openExternal)
+            require('electron').shell.openExternal(url)
+          })
+        }
+      }
+    }])
+})()
```
Comment (spresnal) [inline client/app/src/directives/open-external.directive.js]: This is clean. Nice!

File: client/app/src/components/layout/main-navbar.html (hunk 2 above)
Comment (j-a-m-l) [reply client/app/src/components/layout/main-navbar.html]: Thanks!

File: client/app/index.html (hunk 1 above)
Comment (j-a-m-l) [reply client/app/index.html]: I've changed the rest of the files to kebab-case.

**Pull request merged**